src/api/java/com/minecolonies/api/research/effects/IResearchEffectManager.java

```diff
@@ -17,6 +17,13 @@
      */
     <W extends IResearchEffect<?>> W getEffect(final String id, @NotNull final Class<W> type);
 
+    /**
+     * Check if the research effect exists, and if so, if it has been completed.
+     * @param  id   the id of the effect
+     * @return null if not existing, false if incomplete, true if complete.
+     */
+    Boolean getEffect(final String id);
```

Comment:
this seems in confusing/conflicting with the function right above. Perhaps hasEffect or effectExists as a name?